string_utils/fms_string_utils_binding.c

```diff
@@ -162,3 +162,25 @@ char* fms_find_my_string_binding(char** arr, int *n, char *find_me, int *np)
   strcpy(string_p, string);
   return string_p;
 }
+
+/*!
+ * @brief Finds the number of unique strings in an array
+ * @param[in]  arr Array of strings
+ * @param[in]  n   Size of the array
+ * @return Number of unique strings in an array
+ */
+int fms_find_unique(char** arr, int *n)
+{
+  int i; // For loops
+  int nfind; // Number of unique strings in an array
+  int * ids = calloc(*n, sizeof(int)); // Array of integers initialized to 0
+
+  fms_sort_this(arr, n, ids);
```

Comment:
@uramirez8707 
The fms_sort_this function uses qsort function from stdlib.h. I was able to look at the code and web comments and it seems to me that it uses the known/published tricks to avoid the worst (O(N^2)) cases. But I am not 100% sure. To be closer to sure, one can check if he array is already sorted or reverse is sorted. This is fast (O(N)) in compariosn to qsort. One can also test and determine the performance as functin of size of input, with test data in sorted order, reverse sorted order, and random order.